rig-core/src/providers/cohere.rs

```diff
@@ -217,6 +217,11 @@ impl embeddings::EmbeddingModel for EmbeddingModel {
         if response.status().is_success() {
             match response.json::<ApiResponse<EmbeddingResponse>>().await? {
                 ApiResponse::Ok(response) => {
+                    tracing::info!(target: "rig",
+                        "Cohere embeddings billed units: {:?}",
+                        response.meta.map(|meta| meta.billed_units)
```

Comment:
Same here, instead of using the debug output, just convert the `billed_units` to a string (with `n/a` as default if `billed_units` is `None`)